lib/trie/child_storage.go

```diff
@@ -12,6 +12,8 @@ import (
 // ChildStorageKeyPrefix is the prefix for all child storage keys
 var ChildStorageKeyPrefix = []byte(":child_storage:default:")
 
+const ErrChildTrieDoesNotExists = "child trie does not exist at key"
```

Comment:
```suggestion
const ErrChildTrieDoesNotExist = "child trie does not exist at key"
```

also, this isn't an error type but just a string, do we need to export it or call it `Err`?

Reply:
changed it to error type in later commits